static/templates/settings/custom-user-profile-field.handlebars

```diff
@@ -4,16 +4,22 @@
         <textarea maxlength="500" class="custom_user_field_value">{{ field_value }}</textarea>
     {{else}}
         {{#if is_choice_field }}
-        <select class="custom_user_field_value">
-            {{#unless field_value}}
-            <option></option>
-            {{/unless}}
-            {{#each field_choices}}
-            <option value="{{ this.value }}" {{#if this.selected}}selected{{/if}}>{{ this.text }}</option>
-            {{/each}}
-        </select>
+            <select class="custom_user_field_value">
+                {{#unless field_value}}
+                <option></option>
+                {{/unless}}
+                {{#each field_choices}}
+                <option value="{{ this.value }}" {{#if this.selected}}selected{{/if}}>{{ this.text }}</option>
+                {{/each}}
+            </select>
         {{else}}
-        <input class="custom_user_field_value" type="{{ field_type }}" value="{{ field_value }}" maxlength="50" />
+            {{#if is_user_field }}
+            <div class="pill-container">
+                <div class="input" contenteditable="true"></div>
+            </div>
+            {{else}}
+            <input class="custom_user_field_value" type="{{ field_type }}" value="{{ field_value }}" maxlength="50" />
+            {{/if}}
```

Comment:
@shubhamdhama Do you have suggestions on how this `if..else` can be reduced to simple logic?

Reply:
Sorry for missing this, @YJDave  this looks right but have you tried using `{{else if}}` here,(which is available from handlebars 3.0 and I guess we are using handlebars>4).
More info here https://stackoverflow.com/a/10884049/7418550

Reply:
@shubhamdhama Thanks! Updated handlebar with `{{else if}}`.